tools/cli/go-whisk-cli/commands/api.go

```diff
@@ -1421,6 +1448,21 @@ func parseSwaggerApiV2() (*whisk.Api, error) {
         return nil, whiskErr
     }
 
+    // Check if this swagger is in JSON or YAML format
+    isYaml := strings.HasSuffix(flags.api.configfile, "yaml") || strings.HasSuffix(flags.api.configfile, "yml")
```

Comment:
.yaml and .yml are common YAML file name extensions
https://en.wikipedia.org/wiki/YAML